common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/responses/signup/SignUpChallengeApiResponse.kt

```diff
@@ -41,17 +41,28 @@ import java.net.HttpURLConnection
 class SignUpChallengeApiResponse(
     @Expose override var statusCode: Int,
     correlationId: String,
+    @SerializedName("continuation_token") val continuationToken: String?,
     @Expose @SerializedName("challenge_type") val challengeType: String?,
     @SerializedName("challenge_target_label") val challengeTargetLabel: String?,
     @Expose @SerializedName("code_length") val codeLength: Int?,
     @Expose @SerializedName("binding_method") val bindingMethod: String?,
     @Expose @SerializedName("interval") val interval: Int?,
     @Expose @SerializedName("challenge_channel") val challengeChannel: String?,
-    @SerializedName("continuation_token") val continuationToken: String?,
     @SerializedName("error") val error: String?,
     @SerializedName("error_description") val errorDescription: String?,
 ) : IApiResponse(statusCode, correlationId) {
 
+    override fun toUnsanitizedString(): String {
+        return "SignInChallengeApiResponse(statusCode=$statusCode, " +
+                "correlationId=$correlationId, challengeType=$challengeType, " +
+                "bindingMethod=$bindingMethod, challengeTargetLabel=$challengeTargetLabel, " +
+                "challengeChannel=$challengeChannel, codeLength=$codeLength, interval=$interval, " +
+                "error=$error, errorDescription=$errorDescription)"
+    }
+
+    override fun toString(): String = "SignInChallengeApiResponse(statusCode=$statusCode, " +
+            "correlationId=$correlationId"
```

Comment:
nit: Missing bracket and some parameters. ("challengeChannel=$challengeChannel, codeLength=$codeLength, interval=$interval")